src/main/java/eu/darkbot/api/entities/utils/Area.java

```diff
@@ -0,0 +1,15 @@
+package eu.darkbot.api.entities.utils;
+
+import java.awt.Shape;
+
+public interface Area extends Shape {//?
+
+    AreaType getAreaType();
+
+    enum AreaType {
+        RECTANGLE,
+        CIRCULAR,
+        TRIANGLE,
+        UNKNOWN
+    }
```

Comment:
```suggestion
```
It's up to the implementator to decide this.

For Area just consider it a "place", it's shape isn't publicly known. 
Probably the most abstract methods (that will fill in the gaps required to be used properly) should be methods to see if:
 - It `contains` a point
 - If a line `intersects` this area (could default to intersecting with the bounds)
 - `getBounds` returning a rectangle that contains all inner points
 - Possibly a `getPoints()` that returns points arround this area, used for navigation.